site/class-1-load/IIYAMA12/scripts/dataLoader.js

```diff
@@ -0,0 +1,76 @@
+if (d3) {
```

Comment:
Could you rename this file to `index.js`? Just to keep our repo clean and simple!